win32.js

```diff
@@ -44,8 +44,14 @@ module.exports = {
       if (opts.appCopyright) {
         rcOpts['version-string'].LegalCopyright = opts.appCopyright
       }
+      
+      // add "asInvoker" to the built manifest by default
+      rcOpts['requested-executation-level'] = 'asInvoker'
+      if (opts.win32exeLevel) {
```

Comment:
> The tests that I've seen only really test if the options actually get set, rather than test that the option setting did anything.

Yeah, I'd prefer these tests. There are too many integration tests as it is, which is why the testsuite takes so long to run. I just want tests that assert that the default and custom values get set appropriately.